templates/gitlab.rb.j2

```diff
@@ -322,7 +322,7 @@ redis_exporter['enable'] = false
 # gitlab_rails['backup_pg_schema'] = 'public'
 
 ###! The duration in seconds to keep backups before they are allowed to be deleted
-# gitlab_rails['backup_keep_time'] = 604800
+gitlab_rails['backup_keep_time'] = 1

```

Comment:
I believe setting this to 1 will have the effect of subsequent runs of the GitLab backup command removing all old backups each time. Is this what you intended?

Reply:
Yup that's the intention. 

Layer-backup will run the gtilab backup and then tar the results with the config files on each run. The final backup location is pruned based on a config value for the number of backups you want to keep. No reason to keep multiple copies of the database backup in each of the final tar files.

Reply:
OK makes sense